docs/controller-skills/callsigns.md

```diff
@@ -18,9 +18,9 @@ This page lists some of the less-known aircraft callsigns that are routinely use
 | AWK  | Airwork           | Airwork                | Yes                   |
 | CAC  | Canterbury        | Canterbury Aero Club   | Unsure                |
 | CAP  | Capital           | Helilink               | Yes                   |
-| CFL  | Careflight        | Unsure                 | Unsure                |
+| CFL  | Careflight        | Unsure                 | Yes                   |
 | CGD  | Coast Guard       | Coast Guard            | Yes                   |
-| CVA  | Chatham           | Air Chathams           | Yes                   |
+| CVA  | Chathams          | Air Chathams           | Yes                   |
```

Comment:
As per [CAA page ](https://www.aviation.govt.nz/aircraft/aircraft-registration/operator-call-signs/) it does not have an 's'.